### PR TITLE
add copy to clipboard functionality

### PR DESCRIPTION
**Pull request by alitorbati**

### CODE REVIEW

File: copy.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+(function() {
```
Comment (notwaldorf) [inline copy.js]: TBH I wouldn't use a separate file for this, to minimize downloads. Just use the existing `app.js`

File: app.js [hunk 2]
```diff
@@ -36,6 +36,8 @@
   webfontOutput.addEventListener('blur', changeText);
   webfontOutput.addEventListener('focus', clearText);
 
+  updateClipboardButton()
```
Comment (notwaldorf) [inline app.js]: `;` at the end, please

File: app.js [hunk 3]
```diff
@@ -99,6 +101,14 @@
 
   function updateStyle(name, element, value) {
     document.getElementById(element).style[name] = value;
+    updateClipboardButton()
```
Comment (notwaldorf) [inline app.js]: `;`

File: index.html [hunk 4]
```diff
@@ -41,6 +41,9 @@
       });
     }
   </script>
+
```
Comment (notwaldorf) [inline index.html]: I'd move this to the end of the file, right before the `app.js` import

**Review by notwaldorf — CHANGES_REQUESTED** [review]

**Comment by notwaldorf** [comment]
Looking good! I'll style the button afterwards, I just left some tiny comments in the meantime :)

**Review by notwaldorf — APPROVED** [review]

**Comment by notwaldorf** [comment]
Wooo! Thanks so much!

**Comment by alitorbati** [comment]
happy to contribute!

**Pull request merged**